pkg/packagekit/updates.less

```diff
@@ -4,6 +4,11 @@ table.listing-ct thead th:last-child {
 }
 
 tr.listing-ct-item {
+    vertical-align: baseline;
```

Comment:
I find it more pleasing, especially on mobile, to align the data to the baseline of the first row. (Basically: top aligned instead of middle aligned.)

What do you think? We can remove this (and the vertical-align middle below for the expander) if you disagree.